docs/src/scss/uswds-site.scss

```diff
@@ -1,6 +1,15 @@
 @use "uswds-core" as *;
 @use 'sass:math';
 
+table {
```

Comment:
we really shouldn't be adding any extra styles here. wanna look through the uswds docs to see what we're missing

Reply:
removed - was there in original testing / will revisit css/styles in doc site in another ticket.